src/client/components/HealthCareApp.jsx

```diff
@@ -106,10 +106,11 @@ class HealthCareApp extends React.Component {
           throw new Error(response.statusText);
         }
         response.json().then(data => {
+          console.log(data.response);
           this.props.onUpdateSubmissionStatus('applicationSubmitted', data);
           this.props.onCompletedStatus(path);
-          this.props.onUpdateSubmissionId(data.formSubmissionId);
-          this.props.onUpdateSubmissionTimestamp(data.timeStamp);
+          this.props.onUpdateSubmissionId(data.response.formSubmissionId);
```

Comment:
Yes it was, and yes it was weird because it wasn't visible in the store so I am not quite sure what time it was pulling into the view.